### PR TITLE
Refactoring Rotkelchen into Rotkehlchen

### LINKED ISSUE #6: Typo for the main python module
This folder https://github.com/rotkehlchenio/rotkehlchen/tree/master/rotkelchen should be called `rotkehlchen` or you may need to rename the project entirely.
After all, YOU pick the name, whether it is german or not :)

### PR DESCRIPTION
**Pull request by chevdor**
Fix #6

### CODE REVIEW

File: rotkehlchen/csv_exporter.py [hunk 1]
```diff
@@ -329,7 +329,7 @@ def add_tx_gas_cost(
             timestamp=timestamp,
         )
 
-    def create_files(self, out_dir='/home/lefteris/.rotkelchen/CSV/'):
+    def create_files(self, out_dir='/home/lefteris/.rotkehlchen/CSV/'):
```
Comment (LefterisJP) [inline rotkehlchen/csv_exporter.py]: ops that default should go away. Will handle it soon.

File: tests/test_accounting.py [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-from rotkelchen.accounting import Accountant
-from rotkelchen.history import PriceHistorian, trades_from_dictlist
-from rotkelchen.errors import CorruptData
+from rotkehlchen.accounting import Accountant
+from rotkehlchen.history import PriceHistorian, trades_from_dictlist
+from rotkehlchen.errors import CorruptData
 
 
 def init_accounting_tests(history_list, margin_list, start_ts, end_ts):
-    price_historian = PriceHistorian('/home/lefteris/.rotkelchen')
```
Comment (LefterisJP) [inline tests/test_accounting.py]: This too I should change. But well ... tests need to be reworked with proper fixtures to set data directory, historical prices e.t.c. properly.

File: rotkehlchen/csv_exporter.py (hunk 1 above)
Comment (chevdor) [reply rotkehlchen/csv_exporter.py]: Added #12 to track this one down

File: tests/test_accounting.py (hunk 2 above)
Comment (chevdor) [reply tests/test_accounting.py]: Added #12 to track this one down

**Comment by chevdor** [comment]
Thanks for the fast merge and the public mention :)

**Pull request merged**